src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties

```diff
@@ -1,8 +1,8 @@
 #X-Generator: crowdin.com
 # Strings used by the add-on
 tips.desc=\u63d0\u793a\u548c\u6280\u5de7
-tips.button.allTips=All tips
-tips.button.nextTip=Next tip
+tips.button.allTips=Alle tips
+tips.button.nextTip=Pr\u00f3ximo Consejo
```

Comment:
Sorry I missed that. Have you reverted via crowdin or do you need me to do something? I don't mind just point me in type right direction.